float64 timestamp
float64 ch1_value
float64 ch2_value
float64 ch1_unit
float64 ch2_unit
